### PR TITLE
chore: update to Aspect CLI Pro 4.2.2

### PR DESCRIPTION
**Pull request by gregmagolan**

### CODE REVIEW

File: .aspect/cli/plugins.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Plugins fail on CI: https://github.com/aspect-build/rules_js/actions/runs/3451833892/jobs/5761331944
+# ```
+# Error: failed to run test command: rpc error: code = ResourceExhausted desc = grpc: received message larger than max (7366885 vs. 4194304)
+# Error: failed to run test command: rpc error: code = ResourceExhausted desc = grpc: received message larger than max (7366885 vs. 4194304)
+# Error: failed to run test command: rpc error: code = ResourceExhausted desc = grpc: received message larger than max (8031334 vs. 4194304)
+# Error: failed to run test command: rpc error: code = ResourceExhausted desc = grpc: received message larger than max (8031334 vs. 4194304)
+# ```
+# - name: plugin-aspect-pro
```
Comment (jbedard) [inline .aspect/cli/plugins.yaml:8]: Are these plugins what would be enabled if the file didn't exist?

File: .aspect/cli/plugins.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Plugins fail on CI: https://github.com/aspect-build/rules_js/actions/runs/3451833892/jobs/5761331944
```
Comment (jbedard) [inline .aspect/cli/plugins.yaml:1]: Upgrading caused this? So it's a cli issue?

File: .aspect/cli/plugins.yaml (hunk 1 above)
Comment (gregmagolan) [reply .aspect/cli/plugins.yaml:8]: I attempted to add the plugins in this PR but there is an issue with the CLI so just left them commented out and created a silo issue for it instead.

File: .aspect/cli/plugins.yaml (hunk 2 above)
Comment (gregmagolan) [reply .aspect/cli/plugins.yaml:1]: The issue exists on both 4.2.1 & 4.2.2 of the CLI. Just adding the plugins breaks.

**Review by jbedard — APPROVED** [review]

**Pull request merged**